htdocs/langs/en_US/webhook.lang

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2022 SuperAdmin 
```

Comment:
Because this module has no user interface, except setup page, to reduce nb of small files not really useful, can you move this entries into the file admin.lang and remove completely this file.